src/VisualStudio/Core/Def/DocumentOutline/DocumentOutlineControl.xaml

```diff
@@ -0,0 +1,144 @@
+                <Setter Property="FontSize" Value="12" />
+            </Style>
+
+            <Style TargetType="TextBox" BasedOn="{StaticResource {x:Static vsshell:VsResourceKeys.ThemedDialogTextBoxStyleKey}}" >
+                <Setter Property="FontFamily" Value="Segoe UI" />
+                <Setter Property="FontSize" Value="12" />
+            </Style>
+        </ResourceDictionary>
+    </UserControl.Resources>
+    <Grid Name="myGrid">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="30" />
+            <RowDefinition Height="23" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+        <Grid Grid.Row="0">
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="28" />
+                <ColumnDefinition Width="28" />
+                <ColumnDefinition Width="28" />
+                <ColumnDefinition Width="28" />
+                <ColumnDefinition Width="28" />
+            </Grid.ColumnDefinitions>
+            <Button Grid.Column="0" Click="CollapseAll" ToolTip="{x:Static local:DocumentOutlineStrings.Document_Outline_Expand_All}">
+                <imaging:CrispImage Moniker="{x:Static imagecatalog:KnownMonikers.CollapseAll}" />
+            </Button>
+            <Button Grid.Column="1" Click="ExpandAll" ToolTip="{x:Static local:DocumentOutlineStrings.Document_Outline_Collapse_All}">
+                <imaging:CrispImage Moniker="{x:Static imagecatalog:KnownMonikers.ExpandAll}" />
+            </Button>
+            <Button Grid.Column="2" Click="SortByName" ToolTip="{x:Static local:DocumentOutlineStrings.Document_Outline_Sort_Name}">
+                <imaging:CrispImage Moniker="{x:Static imagecatalog:KnownMonikers.SortAscending}" />
+            </Button>
+            <Button Grid.Column="3" Click="SortByOrder" ToolTip="{x:Static local:DocumentOutlineStrings.Document_Outline_Sort_Order}">
+                <imaging:CrispImage Moniker="{x:Static imagecatalog:KnownMonikers.SortByNamespace}" />
+            </Button>
+            <Button Grid.Column="4" Click="SortByType" ToolTip="{x:Static local:DocumentOutlineStrings.Document_Outline_Sort_Type}">
+                <imaging:CrispImage Moniker="{x:Static imagecatalog:KnownMonikers.SortByType}" />
+            </Button>
+        </Grid>
+        <TextBox Grid.Row="1" x:Name="searchBox" TextChanged="Search" VerticalContentAlignment="Center">
+            <TextBox.Style>
+                <Style TargetType="TextBox">
+                    <Setter Property="Background" Value="{StaticResource {x:Static vsshell:VsBrushes.ToolWindowBackgroundKey}}" />
+                    <Setter Property="Foreground" Value="LightGray" />
+                    <Setter Property="Padding" Value="3, 0, 0, 0" />
+                    <Setter Property="FontFamily" Value="Segoe UI" />
+                    <Setter Property="FontSize" Value="12" />
+                    <Setter Property="CaretBrush" Value="White" />
```

Comment:
I can't imagine this is correct for all themes?

Reply:
CC:  @ryzngard who has been doing some theming recently

Reply:
If the control does not adapt to themes then it's _fine_ and we can update to theme it. https://github.com/dotnet/roslyn/pull/55971 has work to make it easier to theme our tools/windows. Can happily guide through the process.